deployment_botpress/init-letsencrypt.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+set -o allexport
+source .env.nginx
+set +o allexport
+
+# Check that docker compose command is available
+if ! [ -x "$(command -v docker)" ]; then
+  echo 'Error: docker is not installed.' >&2
+  exit 1
+fi
+
+if ! docker compose version >/dev/null 2>&1; then
+  echo 'Error: docker compose is not available.' >&2
+  exit 1
+fi
+
+domains=("$DOMAIN")
+rsa_key_size=4096
+data_path="./data/certbot"
+email="$MAIL" # Adding a valid address is strongly recommended
+staging=0 # Set to 1 if you're testing your setup to avoid hitting request limits
+
+if [ -d "$data_path/conf/live/$domains" ]; then
+  read -p "Existing data found for $domains. Continue and replace existing certificate? (y/N) " decision
+  if [ "$decision" != "Y" ] && [ "$decision" != "y" ]; then
+    exit
+  fi
+fi
+
+
+if [ ! -e "$data_path/conf/options-ssl-nginx.conf" ] || [ ! -e "$data_path/conf/ssl-dhparams.pem" ]; then
+  echo "### Downloading recommended TLS parameters ..."
+  mkdir -p "$data_path/conf"
+  curl -s https://raw.githubusercontent.com/certbot/certbot/master/certbot-nginx/certbot_nginx/_internal/tls_configs/options-ssl-nginx.conf > "$data_path/conf/options-ssl-nginx.conf"
+  curl -s https://raw.githubusercontent.com/certbot/certbot/master/certbot/certbot/ssl-dhparams.pem > "$data_path/conf/ssl-dhparams.pem"
+  echo
+fi
+
+echo "### Creating dummy certificate for $domains ..."
+path="/etc/letsencrypt/live/$domains"
+mkdir -p "$data_path/conf/live/$domains"
+docker compose run --rm --entrypoint "\
+  openssl req -x509 -nodes -newkey rsa:$rsa_key_size -days 1\
+    -keyout '$path/privkey.pem' \
+    -out '$path/fullchain.pem' \
+    -subj '/CN=localhost'" certbot
+echo
+
+echo "### Starting nginx ..."
+docker compose -f docker-compose.yml -p aaq-stack up --force-recreate -d nginx
```

Comment:
Wouldn't nginx and certbot names clash with those in the core docker compose if we use the same project name `aaq-core`?

Should we change the project name to `botpress`, or rename the `nginx` service to `botpress_nginx` and `certbot` to `botpress_cerbot` ?

Reply:
Yeah I didn't catch this. Fixing.